.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: "2.1"
+jobs:
+  plan:
+    docker:
+      - image: hashicorp/terraform:0.11.14
+    steps:
+      - checkout
+      - run:
+          name: Set up Terraform
+          # https://discuss.circleci.com/t/create-separate-steps-jobs-for-pr-forks-versus-branches/13419/2
+          command: |
+            if [[ $(echo "$CIRCLE_BRANCH" | grep -c "pull") -gt 0 ]]; then
+              echo "This is from a fork."
+              terraform init -backend=false -input=false terraform
+            else
+              terraform init -backend-config=terraform/backend.tfvars -input=false terraform
+            fi
+      - run:
+          name: Validate Terraform config
+          command: |
+            if [[ $(echo "$CIRCLE_BRANCH" | grep -c "pull") -gt 0 ]]; then
+              echo "This is from a fork."
+              terraform validate terraform
+            else
+              terraform plan -input=false terraform
+            fi
```

Comment:
If we end up having to repeat this config in a bunch of places, might make sense to [create an orb](https://circleci.com/docs/2.0/orb-intro/) for it (later).